ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -211,6 +211,13 @@ then
 	spl06 -X -a 0x77 start
 fi
 
+# SPA06 sensor external I2C
+if param compare -s SENS_EN_SPA06 1
+then
+	spa06 -X start
+	spa06 -X -a 0x77 start
```

Comment:
Why handle 0x77 here?

Reply:
Just copied SPL06. Would you do it somewhere else?

Reply:
@dagar let me know what you want me to change here, otherwise I'll merge as is.